### ISSUE TITLE
Improve DenseBlock

### ISSUE THREAD

**Issue by jhfjhfj1** [keras-team/autokeras#651]
### Feature Description

Current DenseBlock is very simple, we also want to search for the batchnorm, dropout, activation.

**Comment by torokati44** [comment]
DenseBlock still always uses ReLU activation, it is hardcoded.
